### ISSUE TITLE
Break out sentences into own repo

### ISSUE THREAD

**Issue by phirework** [common-voice/common-voice#2992]
The repo is getting huge and this unnecessarily slows down new builds. As new sentences are read only once per release, I'm going to break them out into their own repo and access remotely.

**Comment by phirework** [comment]
Good call on coordinating, I'll make sure that's part of the update. 

> Bonus points for some renaming to reflect that it won't be the Common Voice repo anymore.

I'm not sure what you mean? It'll still be the CV sentences repo and it'll probably still live in the CV Github org.

**Comment by MichaelKohler** [comment]
> I'm not sure what you mean? It'll still be the CV sentences repo and it'll probably still live in the CV Github org.

Yeah, good point. Guess it would be overkill to go for "COMMON_VOICE_SENTENCES_REPO" etc.. :)

**Comment by moz-bozden** [comment]
This is not relevant anymore because we have the sentence collector integrated and uses the DB directly.
The sentences in `server/data/<lc>/*.txt` only help in development for not - until we create seeders.
Closing.
